docs/events/glides/com.mbta.ctd.glides.operator_signed_in.v1.mdx

```diff
@@ -36,6 +36,7 @@ How the operator signed that they were fit for duty.
 Fields:
 
 - `type` (string): How the the operator made their signature.
+- `version` (integer): The version of the sign-in text the operator agreed to.
```

Comment:
question: would it be more useful to store this as a string, or are we confident that we'll never have versions like "1.1" or "1a"?

Reply:
The Glides database keeps it as an integer, and we don't do any kind of semver or anything on the text, it's just if the text ever changes the number goes up. I think an integer is fine.